app/views/tags/index.html.erb

```diff
@@ -20,140 +21,19 @@
   { link_text: t('tags.download.download'),
     link_path: '#',
     html_options: { onclick: 'modal_download.open()' } },
-  { link_text: t('tags.upload.upload_csv'),
+  { link_text: "Upload",
```

Comment:
This text should be internationalized; look in the locales to see how you can get "Upload"